.github/workflows/gh-pages-deployment.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - redat97/issue24
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: yarn
+          cache-dependency-path: 'ai-cfia-ia-acia'
```

Comment:
repeated constant can be pulled up to env: directive even in a non-reusable workflow.

Reply:
done